Sources/Share/Views/LikeControl.swift

```diff
@@ -64,10 +64,10 @@ public class LikeControl: UIView {
   /// The foreground color to use for the content of the control.
   public var foregroundColor: UIColor {
     get {
-      return sdkLikeControl.foregroundColor
+      return sdkLikeControl.foregroundColor__deprecated
```

Comment:
Can you explain this change?

Reply:
According to Apple you need to add the __deprecated flag to properties that are deprecated so developer are aware they are using deprecated methods and properties.

Reply:
Trying to catch up here. From a little experimenting it seems that adding the __deprecated flag to the objc header will  mark the swift method as deprecated. Ex:

```swift
/**
  The foreground color to use for the content of the receiver.
 */
@property (nonatomic, strong) UIColor *foregroundColor __deprecated;
```
will generate this Swift header:
```swift
/**
  The foreground color to use for the content of the receiver.
 */
@available(*, deprecated)
open var foregroundColor: UIColor!
```

I wasn't able to find anything about the __deprecated flag that would imply that it works in reverse. Could you point me to what you read so I can get caught up? All I was able to find was this post from a while back. https://forums.developer.apple.com/thread/5244 Also, Thanks for helping out with this!

Reply:
Looks like this change is causing the build to fail for `FacebookShare`. There's also new warnings for missing nullability annotations that are treated as errors.